src/homepage/App.vue

```diff
@@ -10,7 +10,7 @@
     name: 'app'
   };
 </script>
-<style lang="scss">
+<style lang="scss" rel="stylesheet/scss">
```

Comment:
Some IDEs need `rel` to tell them this is `scss`, or they will warn the syntax error.